sys-fs/bees/bees-9999.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit multilib linux-info
+
+DESCRIPTION="Best-Effort Extent-Same, a btrfs dedup agent"
+HOMEPAGE="https://github.com/Zygo/bees"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/kakra/bees.git"
+	EGIT_BRANCH="integration"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/Zygo/bees/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+PATCHES="${FILESDIR}/v0.5-gentoo_build.patch"
+
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="tools test"
+
+RDEPEND=">=sys-apps/util-linux-2.30.2"
+DEPEND="$RDEPEND
+	>=sys-fs/btrfs-progs-4.1
+	|| ( dev-python/markdown dev-python/markdown2 )
```

Comment:
If the only reason for this dependency is to create `README.html`, then I'd suggest patching it out. It's not like Gentoo users prefer HTML versions of documents over plaintext, and installing the same content twice is just waste of space.

Reply:
I'm going to patch that out. Hang on.

Reply:
I added a second patch to remove README.html from the build system.